package/MDAnalysis/lib/distances.py

```diff
@@ -114,29 +114,22 @@ def _run(funcname, args=None, kwargs=None, backend="serial"):
 
 from c_distances_openmp import OPENMP_ENABLED as USED_OPENMP
 
-
 def _box_check(box):
     """Take a box input and deduce what type of system it represents based
     on the shape of the array and whether all angles are 90.
 
-    Parameters
-    ----------
-    box : array
-        Box information of unknown format.
-
-    Returns
-    -------
-    boxtype : str
-        * ``ortho`` orthogonal box
-        * ``tri_vecs`` triclinic box vectors
-        * ``tri_box`` triclinic box lengths and angles
-
-    Raises
-    ------
-    TypeError
-        If box is not float32.
-    ValueError
-        If box type not detected.
+    :Arguments:
```

Comment:
This looks like an error that must have happened when you merged the develop branch. If you split this PR we can get rid of this as well and it will also make it easier to remove the big test files from the history, I would remove them anyway before the merge because otherwiese they would be stuck in the history and we would have to download them everytime with a fresh checkout.